Dockerfile

```diff
@@ -46,11 +46,13 @@ ARG ATLAS_VERSION=latest
 ENV ATLAS_VERSION=${ATLAS_VERSION}
 RUN curl -sSf https://atlasgo.sh | sh
 
+FROM docker:27.3.1-cli-alpine3.20 as docker
+
 FROM alpine:3.20
+ENV ATLAS_KUBERNETES_OPERATOR=1
 WORKDIR /
-COPY --from=builder /workspace/manager .
 COPY --from=atlas /usr/local/bin/atlas /usr/local/bin
-RUN chmod +x /usr/local/bin/atlas
-ENV ATLAS_KUBERNETES_OPERATOR=1
+COPY --from=docker /usr/local/bin/docker /usr/local/bin
```

Comment:
why not simply run `apk add docker` ?

Reply:
I want to version the `docker` cli, also need only one binary. Nothing else.

Reply:
btw, I'm not sure if we should bundle docker-cli with the default image, or create a separated tag for `dind` support

Reply:
The second option sounds reasonable